src/util/Handlers.js

```diff
@@ -0,0 +1,51 @@
+/*!
+ * Copyright (c) 2018-Present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+define(['util/Logger'], function (Logger) {
+
+  var Handlers = {};
+
+  Handlers.defaultSuccessTokenHandler = function(tokenManager, keys = {}) {
```

Comment:
Looking ahead to #487, could this also be used to handle tokens received via `success(res)` in a silent login flow?

Reply:
Correct - with the assumption that a new interface is exposed.

We run into this problem currently where exposed Widget APIs expect callbacks to handle `async` methods (Why this is treated as an `async` function is another thing to look into).